CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+## [1.6.0] - 2021-12-15
+### Added
+- Added optional <optgroup> label to DropdownItem
```

Comment:
you also need the link of the diff at the bottom of the file

Reply:
@fleonard added